src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,53 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+  background-size: cover;
+  background-position: center;
+  transition: transform 0.3s;
+
+  @include hover(transform, scale(1.2));
+
+  @include onTablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    position: relative;
+    background-image: url(../images/Icons/icon-phone.svg);
```

Comment:
I believe the " is missed. this is happening in other parts of the code as well

`background-image: url("../images/Icons/icon-phone.svg");`